qutebrowser/utils/version.py

```diff
@@ -46,7 +46,7 @@
 
 import qutebrowser
 from qutebrowser.utils import log, utils, standarddir, usertypes, qtutils
-from qutebrowser.misc import objects
+from qutebrowser.misc import objects, sql
```

Comment:
This is causing `earlyinit.py` to already fail without SQL support:

```
Traceback (most recent call last):
  File "/usr/lib/python3.4/", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/", line 85, in _run_code
    exec(code, run_globals)
  File "/home/florian/qutebrowser/qutebrowser/__main__.py", line 29, in <module>
    sys.exit(qutebrowser.qutebrowser.main())
  File "/home/florian/qutebrowser/qutebrowser/", line 174, in main
    earlyinit.earlyinit(args)
  File "/home/florian/qutebrowser/qutebrowser/misc/", line 415, in earlyinit
    check_qt_version(backend)
  File "/home/florian/qutebrowser/qutebrowser/misc/", line 259, in check_qt_version
    from qutebrowser.utils import qtutils, version
  File "/home/florian/qutebrowser/qutebrowser/utils/", line 49, in <module>
    from qutebrowser.misc import objects, sql
  File "/home/florian/qutebrowser/qutebrowser/misc/", line 25, in <module>
    from PyQt5.QtSql import QSqlDatabase, QSqlQuery
ImportError: No module named 'PyQt5.QtSql'
```

I'll fix this in `master` by not importing `utils.version` in `earlyinit`, but if you want to check the "no SQL support installed" thing mentioned above, you might need to temporarily remove the import.